versioned_docs/version-v3.0.x/extend/extend-apiml/api-mediation-infinispan.md

```diff
@@ -28,36 +28,52 @@ For more information on Infinispan replication and how to configure a replica in
+  The default value is `data`. To run the Caching Service in HA, ensure that you apply the following configuration conditions:
 
+   - The value should be the same for each instance.
+   - The location should point to non-shared filesystem. Each instance requires unique storage.
+   - For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
 
-* **`zowe.components.caching-service.storage.infinispan.persistence.indexLocation`**
+
+* **zowe.components.caching-service.storage.infinispan.persistence.indexLocation**
 
   The path where the service keeps its index data for the Infinispan Soft-Index Cache Store. 
-  The default value is `index`. If you run the Caching Service in HA and the instances use the same filesystem, you have to specify a different value of the index property for each instance. For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
+  The default value is `index`. To run the Caching Service in HA, ensure that you apply the following configuration conditions:
+
+  - The value should be the same for each instance.
+  - The location should point to non-shared filesystem. Each instance requires unique storage.
+  - For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
 
 
-* **`zowe.components.caching-service.storage.infinispan.jgroups.port`**
+* **zowe.components.caching-service.storage.infinispan.jgroups.port**
 
-  (OPTIONAL)The default value is `7098`. The port number used by Infinispan to synchronise data among cahing-service instances.
+  (Optional) The default value is `7600`. The port number is used by Infinispan to synchronise data among Caching Service instances.
 
-* **`zowe.components.caching-service.storage.infinispan.jgroups.host`**
+  :::note
+  We recommend you define this value to avoid potential problems or errors in future Zowe upgrades, for example 
+  from version 2.x through v3.1 to v3.2 and newer versions.
+  :::
 
-  (OPTIONAL)The default value is taken from zowe hostname. The hostname used by Infinispan to synchronise data among cahing-service instances. 
+* **zowe.components.caching-service.storage.infinispan.jgroups.host**
 
+  (Optional) The default value is taken from zowe hostname. The hostname used by Infinispan to synchronise data among Caching Service instances. 
 
-* **`zowe.components.caching-service.storage.infinispan.keyExchange.port`**
+* **zowe.components.caching-service.storage.infinispan.keyExchange.port**
 
-  (OPTIONAL)The default value is `7118`. The port number used by Infinispan to exchange encryption key among cahing-service instances.
+  (Optional) The default value is `7601`. The port number is used by Infinispan to exchange encryption key among Caching Service instances.
```

Comment:
This one is not defined in https://github.com/zowe/zowe-install-packaging/blob/v3.x/staging/files/defaults.yaml
Default is 7118